src/main/java/com/ibm/watson/modelmesh/ModelMeshApi.java

```diff
@@ -716,12 +727,24 @@ public void onHalfClose() {
                                         balancedMetaVal, headers, reqMessage, allRequired);
                             }
                         } finally {
+                            try {
+                                payloadProcessor.processRequest(new Payload(payloadId, modelId, String.valueOf(isVModel),
```

Comment:
The reason that the resoled `modelId` might be important is that it corresponds to the actual version of the underlying model being hit by that request. Requests sent to the same `vModelId` could hit different models over time, and I was assuming that it would be important to know which version of the model certain metrics correspond to.

I think it's ok to get the PR in without it though, because it won't be a breaking change to add as a follow-on.

Reply:
detecting which version of a model is being used is definitely important for things like detecting bias over different versions of a same model. I'll create a new issue for this, so that we can address this multiple version thing on top of this PR.